sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py

```diff
@@ -1492,6 +1492,121 @@ def __init__(
         except StorageErrorException as error:
             process_storage_error(error)
 
+    @distributed_trace_async
+    async def upload_pages_from_url(self, source_url,  # type: str
+                              range_start,
+                              range_end,
+                              source_range_start,
+                              source_content_md5=None,
+                              source_if_modified_since=None,
+                              source_if_unmodified_since=None, source_if_match=None, source_if_none_match=None,
+                              lease=None, if_sequence_number_lte=None, if_sequence_number_lt=None,
+                              if_sequence_number_eq=None, if_modified_since=None, if_unmodified_since=None,
+                              if_match=None, if_none_match=None, timeout=None
+                              ):
```

Comment:
Same comment as for `append_block_from_url`, everything after `source_content_md5` can be combined in **kwargs

Reply:
Thanks for pointing this out!
While I'm still curious how you usually decide if you want to put a parameter in **kwargs or not